clouddrift/haversine.py

```diff
@@ -1,3 +1,9 @@
+"""
+This module provides various Haversine functions to compute distances and
```

Comment:
kinda makes me think that this module could be folded in the sphere module?

Reply:
but not now

Reply:
Yes, I think that would be appropriate.